src/styles/blocks/catalog.scss

```
@@ -0,0 +1,31 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  justify-content: center;
+  gap: $gap-size $gap-size;
+
+  max-width: 944px;
+  margin: 0 auto;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $card-width;
+    height: $card-height;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: 200px;
+  }
+}
```

you have variable $card-width.
Use it)
```suggestion
    grid-template-columns: repeat(3, $card-width);
  }

  @media (max-width: 767px) {
    grid-template-columns: repeat(2, $card-width);
  }

  @media (max-width: 487px) {
    grid-template-columns:  $card-width;
  }
}
```